plugins/kubernetes/app/models/kubernetes/namespace.rb

```diff
@@ -51,10 +51,9 @@ def parsed_template
     end
 
     def validate_template
-      unless parsed_template.is_a?(Hash)
-        return errors.add :template, "needs to be a Hash"
-      end
-      errors.add :template, "needs metadata.labels.team" unless parsed_template.dig("metadata", "labels", "team")
+      return errors.add :template, "needs to be set" if template.blank?
+      return errors.add :template, "needs to be a Hash" unless parsed_template.is_a?(Hash)
+      return errors.add :template, "needs metadata.labels.team" unless parsed_template.dig("metadata", "labels", "team")
```

Comment:
notice the symmetry :trollface:

Reply:
I'd prefer UI to be populated with valid empty hash.